sweagent/agent/models.py

```
@@ -25,7 +27,80 @@
 
 logger = get_logger("api_models")
 
-_MAX_RETRIES = keys_config.get("SWE_AGENT_MODEL_MAX_RETRIES", 10)
+# TODO: [PRO-848] Configure retries
+_MAX_RETRIES = keys_config.get("SWE_AGENT_MODEL_MAX_RETRIES", 0)
```

If we allow retries, the model is allowed to fail silently.